### ISSUE TITLE
No Units for DLIQ in english units

### ISSUE THREAD

**Issue by ianhbell** [usnistgov/REFPROP-issues#456]
If ``DLIQ`` is provided as output, then the ``hUnits`` is always `-`.
Example:
```
import ctREFPROP.ctREFPROP as c, os
RP = c.REFPROPFunctionLibrary(os.environ['RPPREFIX'])
print(RP.RPVersion())
r1 = RP.REFPROPdll('R513A', 'QT', 'DLIQ', RP.MOLAR_BASE_SI, 0, 0, 0, 255, [1.0])
print(r1.Output[0:2], r1.hUnits)
```
yielding
```
10.0.0.02
array('d', [11794.217228486597, -9999990.0]) -
```

**Comment by EricLemmon** [comment]
This appears to have been fixed in the latest alpha version, 10.0.1.8, see picture below.

![image](https://user-images.githubusercontent.com/42392984/134759685-b1d08a04-8017-4b33-806b-fd86b957dea6.png)

**Comment by ianhbell** [comment]
Fixed in version 10.0.1.3.  Please email ian.bell@nist.gov for copy of development version.
